ngrinder-frontend/build.gradle

```diff
@@ -18,4 +18,5 @@ task webpack(dependsOn: "npmInstall", type: NodeTask) {
 task checkES5(type: NpxTask) {
     command = "es-check@$esCheckModuleVersion"
     args = ["es5", "../ngrinder-controller/build/classes/main/static/js/app.js", "--verbose"]
+    ignoreExitValue = true
```

Comment:
IE11 is officially deprecated from 2022.06.15 so, how about we stop supporting IE too?

Reply:
Yep. stop supporting this.